wscript

```diff
@@ -319,6 +319,19 @@ iconv support use --disable-iconv.",
         'func': check_pkg_config('smbclient'),
         'default': 'disable',
         'module': 'input',
+    }, {
+        'name': '--libtasn1',
+        'desc': 'GnuTLS',
+        'func': check_pkg_config('libtasn1'),
+        'default': 'disable',
+        'module': 'input',
+    }, {
+        'name': '--libdsm',
+        'desc': 'Samba & cifs support via libdsm',
+        'deps': 'libtasn1',
```

Comment:
Can you check if both libsmbclient and libdsm work in the same binary? With a quick look the function prefixes are different (`smbc_` vs `smb_`), but if it fails then we need to make libsmbclient and libdsm depend on the other one not being enabled :) .

Reply:
exactly, only one of them should working for a certain build.